### PR TITLE
(fleet/kuma) deploy a temporary kuma dev monitoring instance 

### PR DESCRIPTION
**Pull request by dtapiacl**
This will be "temporary", yes i know we don't like to use that word around here.

### CODE REVIEW

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Review by jhoblitt — COMMENTED** [review]
When looking at the repo history, this commit message wouldn't be very helpful:
image

File: fleet/lib/kuma/values.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+ingress:
+  enabled: true
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.class: nginx
```
[the remark this answers is not in the document]
Comment (dtapiacl) [reply fleet/lib/kuma/values.yaml:6]: I think it does based on the source code:

https://github.com/dirsigler/uptime-kuma-helm/blob/1e70146809f3f651a00af065a734491facb2caa2/charts/uptime-kuma/templates/ingress.yaml#L30

I will update the code to use that instead.

File: fleet/lib/kuma/values.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    nginx.ingress.kubernetes.io/backend-protocol: HTTP
+    nginx.ingress.kubernetes.io/server-snippet: |
+      proxy_ssl_verify off;
```
[the remark this answers is not in the document]
Comment (dtapiacl) [reply fleet/lib/kuma/values.yaml:9]: Ok, I will clean up line 7.

File: fleet/lib/kuma/values.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    - secretName: kuma-dashboard-ingress-tls
+      hosts:
+        - kuma.dev.lsst.org
+
+
+volume:
+  enabled: true
+  accessMode: ReadWriteOnce
+  size: 4Gi
+  existingClaim: ""
```
[the remark this answers is not in the document]
Comment (dtapiacl) [reply fleet/lib/kuma/values.yaml:27]: I would prefer so if possible, as i did not find a persistence key on the values chart.

File: fleet/lib/kuma/values.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+ingress:
+  enabled: true
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/backend-protocol: HTTP
+    nginx.ingress.kubernetes.io/server-snippet: |
+      proxy_ssl_verify off;
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
```
[the remark this answers is not in the document]
Comment (dtapiacl) [reply fleet/lib/kuma/values.yaml:10]: Probably not. From what I read, I'm going to remove this value as it's optional.

File: fleet/lib/kuma/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dtapiacl) [reply fleet/lib/kuma/values.yaml]: Had to update the code as values uses `className:` now its working.

**Comment by dtapiacl** [comment]
> When looking at the repo history, this commit message wouldn't be very helpful: image

I squashed on to main commit.

**Review by jhoblitt — APPROVED** [review]

**Review by gseriche — APPROVED** [review]

**Pull request merged**